communication/marketing-team/role-handbooks/storytellers.md

```diff
@@ -4,7 +4,7 @@
 
 Upstream Marketing depends on great storytelling to get the message out about our amazing community.
 
-It's no secret that governance and technical documentation can be kind of on the boring side. And that's ok–it's not always there for entertainment purposes. It is about providing valuable information and get you were you need to go. But that said, an important message can be lost without an entertaining messenger. 
+It's no secret that governance and technical documentation can be kind of on the boring side. And that's ok–it's not always there for entertainment purposes. It is about providing valuable information and getting you were you need to go. But that said, an important message can be lost without an entertaining messenger. 
```

Comment:
I think there is still a typo in this sentence. It's should probably be "getting you where you need to go".

Reply:
Thank you for catching that, my bad.